packages/components/src/public/QueryModel/QueryModel.ts

```diff
@@ -320,9 +320,13 @@ export class QueryModel {
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     readonly rows?: { [key: string]: any };
     /**
-     * The total count of rows for the given QueryModel.
+     * The count of rows returned from the query for the given QueryModel.
```

Comment:
Hmm, isn't it more confusing to have both of these on the model? It seems like `totalRows` is something we want to get away from as we already have everything we need expressed in `rowCount`. This seems to proliferate `totalRows` further (outside of `selectRowsDeprecated`).